torch/csrc/jit/mobile/import.cpp

```diff
@@ -27,7 +27,8 @@
 //      ('operators', (('aten::Int', 'Tensor'),)),
 //      ('constants', ()),
 //      ('types', ()),
-//      ('register_size', 2))))
+//      ('register_size', 2)
+//      ('module_debug_info', (top(A).foo(B).forward)))))
```

Comment:
Should we also update this in the comment, that the debug info is in another .pkl file?

Reply:
Thanks for catching this!